torch/testing/_internal/two_tensor.py

```diff
@@ -36,6 +37,7 @@ def __new__(cls, a, b, outer_size=None, outer_stride=None):
         assert a.storage_offset() == b.storage_offset()
         return out
 
+    @_mark_constructor_exportable_experimental
```

Comment:
fyi @albanD @drisspg, new decorator that we're adding, that subclass authors need to opt into in order to support "subclass constructor inside of a graph" for export.

not sure about the name. Although to be fair, once @IvanKobzarev's "base wrapper subclass" design is further along, we can hide the constructor in there for subclass authors that use the base subclass (which will hopefully be all quantization subclasses)

Reply:
Since this is private API, I guess the name is not as important and we can go with as descriptive as possible.
We can always refine the name when this becomes public.

Reply:
I think even in prototype state some good documentation on what exact invariants subclass author is promising to abide by would be helpful